src/vm/method.hpp

```diff
@@ -3277,15 +3310,31 @@ class InstantiatedMethodDesc : public MethodDesc
     }
 #endif // FEATURE_COMINTEROP
 
+    PTR_DictionaryLayout GetDictLayout()
+    {
+        LIMITED_METHOD_DAC_CONTRACT;
+        return RelativePointer<PTR_DictionaryLayout>::GetValueMaybeNullAtPtr(PTR_HOST_MEMBER_TADDR(InstantiatedMethodDesc, this, m_pDictLayout));
+    }
+
+    PTR_MethodDesc GetMethodDesc()
```

Comment:
GetMethodDesc is a confusing name for this method.

This should be merged with `IMD_GetWrappedMethodDesc`. The assert from `IMD_GetWrappedMethodDesc` is good. Also, it can be assumed to be non-null.